developer-docs-site/docs/index.md

```diff
@@ -30,7 +32,7 @@ Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentatio
 </div>
 </div>
 
-## Concepts and guides
+## Learn our concepts and follow the guides
```

Comment:
Suggest change "our" to "Aptos". For a true community approach, I recommend we don't use "our" anywhere in the docs because it wrongly conveys the idea that "we" are Aptos Labs and "they" are everyone else.

Reply:
Ack. Done!